volatility3/framework/layers/registry.py

```diff
@@ -66,13 +66,13 @@ def __init__(self,
             self._hive_maxaddr_non_volatile = self.hive.Storage[0].Length
             self._hive_maxaddr_volatile = self.hive.Storage[1].Length
             self._maxaddr = 0x80000000 | self._hive_maxaddr_volatile
-            vollog.log(constants.LOGLEVEL_VVV, "Setting hive max address to {}".format(hex(self._maxaddr)))
+            vollog.log(constants.LOGLEVEL_VVV, "Setting hive {} max address to {}".format(self.name, hex(self._maxaddr)))
```

Comment:
Sorry to be a pain, but can we swap these to f-strings please?  Now that we're not bound to python 3.5 and I converted the whole base over to f-strings, it would be best to phase these out rather than add in new ones?

Reply:
I'd probably also put this one V higher, since it feels like it might log a lot I'm not sure how often we'll care about it?

Reply:
See d613b384d2cc8b29943bc8e5b7f12af338cb174b